docs_source_files/content/webdriver/page_loading_strategy.de.md

```diff
@@ -247,3 +258,8 @@ fun main() {
 }
   {{< / code-panel >}}
 {{< / code-tab >}}
+
+### Note: 
+For most of the forms that involves testing in a browser, one should always use 
```

Comment:
`For most of the forms that involves testing in a browser`
Same comment as above but for this sentence.

Reply:
hi @diemol, Removed the note for now. Will Cover in a separate PR with solid content.